perf/src/bin/perf_client.rs

```diff
@@ -128,7 +131,11 @@ async fn run(opt: Opt) -> Result<()> {
     let mut transport = quinn::TransportConfig::default();
     transport.initial_max_udp_payload_size(opt.initial_max_udp_payload_size);
 
-    let mut cfg = quinn::ClientConfig::new(Arc::new(crypto));
+    let mut cfg = if opt.nullcipher {
+        quinn::ClientConfig::new(Arc::new(NullCiperClientConfig::new(Arc::new(crypto))))
+    } else {
+        quinn::ClientConfig::new(Arc::new(crypto))
```

Comment:
If the use of crypto is going in this branch, we should also move the initialization of it here?

Reply:
It's used in both branches; the new logic uses it to delegate to TLS for most operations.